src/Files.App/Helpers/FileSecurityHelpers.cs

```diff
@@ -20,29 +14,11 @@ namespace Files.App.Helpers
 	/// </summary>
 	public static class FileSecurityHelpers
 	{
-		public static bool SetOwner(string path, string sid)
-		{
-			SECURITY_INFORMATION secInfo = SECURITY_INFORMATION.OWNER_SECURITY_INFORMATION;
-
-			// Get PSID object from string sid
-			var pSid = ConvertStringSidToSid(sid);
-
-			// Change owner
-			var result = SetNamedSecurityInfo(path, SE_OBJECT_TYPE.SE_FILE_OBJECT, secInfo, pSid);
-
-			pSid.Dispose();
-
-			// Run PowerShell as Admin
-			if (result.Failed)
-			{
-				return Win32API.RunPowershellCommand(
-					$"-command \"try {{ $path = '{path}'; $ID = new-object System.Security.Principal.SecurityIdentifier('{sid}'); $acl = get-acl $path; $acl.SetOwner($ID); set-acl -path $path -aclObject $acl }} catch {{ exit 1; }}\"",
-					true);
-			}
-
-			return true;
-		}
-
+		/// <summary>
+		/// Get file owner.
+		/// </summary>
+		/// <param name="path"></param>
+		/// <returns></returns>
```

Comment:
Is this doc really necessary? It doesn't give much information that one can't get from the function name.

Reply:
Yeah sorry. I will add some more info.

Reply:
Done